modules/apps/29-fee/keeper/events.go

```diff
@@ -9,17 +9,33 @@ import (
 	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
 )
 
-// EmitIncentivizedPacket emits an event so that relayers know an incentivized packet is ready to be relayed
-func EmitIncentivizedPacket(ctx sdk.Context, packetID channeltypes.PacketId, packetFee types.PacketFee) {
+// EmitIncentivizedPacketEvent emits an event containing information on the total amount of fees incentivizing
+// a specific packet. It should be emitted on every fee escrowed for the given packetID.
+func EmitIncentivizedPacketEvent(ctx sdk.Context, packetID channeltypes.PacketId, packetFees types.PacketFees) {
+	var (
+		totalRecvFees    sdk.Coins
+		totalAckFees     sdk.Coins
+		totalTimeoutFees sdk.Coins
+	)
+
+	for _, fee := range packetFees.PacketFees {
+		// only emit total fees for packet fees which allow any relayer to relay
+		if fee.Relayers == nil {
+			totalRecvFees.Add(fee.Fee.RecvFee...)
+			totalAckFees.Add(fee.Fee.AckFee...)
+			totalTimeoutFees.Add(fee.Fee.TimeoutFee...)
```

Comment:
Should we reassign as [`coins.Add(...)` returns `sdk.Coins`](https://github.com/cosmos/cosmos-sdk/blob/main/types/coin.go#L303)? 

```
totalRecvFees = totalRecvFees.Add(...)
```

I think this needs to be done for the singular [`coin.Add()`](https://github.com/cosmos/cosmos-sdk/blob/main/types/coin.go#L105) function. Not 100% sure about `Coins` but may be good practise to keep consistent and handle return value.

Reply:
seems like a good idea to me

Reply:
nice catch :bug: - the tests were using an expected value of "", the lack of reassigning didn't change the variable values